student.rb

```diff
@@ -5,7 +5,14 @@ def initialize(_clasroom, age, name = 'unknown', parent_permission: true)
     @classroom = classroom
   end
 
+  attr_accessor :owner
+
```

Comment:
- [ ] For better code structure, kindly move this declaration to the top of your class